### ISSUE TITLE
Create a multisite website for Houston civic clubs to have rss feeds

### ISSUE THREAD

**Issue by fileunderjeff** [sketch-city/project-ideas#46]
Using Wordpress or Drupal, create a multisite website and an instance for each of the [Houston civic clubs](http://data.houstontx.gov/dataset/civic-clubs). Could include a civic club lookup and registration function. Goal is to provide every civic club with the tools to create an RSS feed. Main site could aggregate all feeds.

**Comment by ardouglass** [comment]
Would there be someone from each of the civic clubs actually creating content for their sites?

**Comment by ardouglass** [comment]
This is a technically simple problem, and I think the real challenge here would be organizing and on-boarding neighborhood association (?) members.

**Comment by fileunderjeff** [comment]
Agreed on its simplicity. I don't think it would be a challenge to get the active civic clubs on board, either. I think we just need a prototype to show what's possible, and how easy it can be.

I wrote this down because I did a research project into emergency management technology about 1.5 years ago, and I had the opportunity to talk with the lead tech guy for Occupy NYC --> Occupy Sandy and how they used simple tech to coordinate resources.

This is a good article that sums up some of it: http://www.forbes.com/sites/deannazandt/2012/11/10/what-sandy-has-taught-us-about-technology-relief-and-resilience/#22f0868cf430

**Comment by fileunderjeff** [comment]
I also think this project can pivot in any number of directions while keeping the same core tech.

**Comment by ardouglass** [comment]
The more I think about this, the more I question how it would be different from something like Nextdoor. I was thinking it would be able to pull in social media posts and save them as posts to the blog as well. That got me thinking about adding Nextdoor as an integration, but they don't have any sort of public API.

So what are a couple use cases for this?

**Comment by fileunderjeff** [comment]
For civic clubs, giving them access to simple tech (e.g. publish an RSS feed) allows them to list events, publish meeting minutes, survey the internet, etc. Many of these orgs do not use social media. The solution would be deliberately simple to reduce any learning curve... hence a WP multisite.

**Comment by fileunderjeff** [comment]
The end user use case is to search and find the appropriate civic club, and to gauge whether it's active.

**Comment by ardouglass** [comment]
This is what I see as a todo list. The technical stuff should probably come second short of throwing together a quick MVP. I just wanted to spit out some long term ideas while they were in my head.

### Non-Technical Stuff
- [ ] Find civic club leadership and see if they have any interest
- [ ] Community organizing???
- [ ] Maybe do a Neighborhood Toolkit workshop like they do in Charlotte where they show neighborhood and other community associations how to use things like Twitter, Facebook Pages, Mailchimp, etc. (I participated in one, and it was actually really cool) - something like this: http://digitalcharlotte.org/events/. I bet they'd be willing to share tips and resources on doing something like this.

### Technical Stuff

**General**
- [ ] Set up server with wp-multisite
- [ ] Build awesome Houston Civic Club theme for all of them to use with some ability to customize like a header image or something. The main page could maybe do some cool stuff like let you choose info to see or subscribe to by map. Remotely [Citygram-ish](https://www.citygram.org/). Sort of.
- [ ] Make sure civic club admins can't break each others or their own blogs (turn off their ability to change plugins and stuff?)

**Custom Post Types**
Because less free form is better when there will be similar and repeatable posts with their own special metadata.
- [ ] Events / calendar - maybe let them link directly to Google Calendar or Facebook events and pull in the data?
- [ ] Meeting minutes - they can specify the meeting date and maybe some other useful metadata?
- [ ] Social media - if they have social media accounts, pull in posts from them and display them and stick them in the RSS feed. Doesn't need an interface other than a settings page to do their OAuth or whatever would be needed to do this.
